share/spice/sublime_packages/footer.handlebars

```diff
@@ -0,0 +1,4 @@
+<div class="tile__update__info one-line">
+  <i class="ddgsi ddgsi-arrow-down"></i>
+  {{unique_installs}} &middot; Updated {{last_modified}}
```

Comment:
Maybe best to check them just to be on the safe side--especially since this is an undocumented API.